### PR TITLE
Add warm start

### PR DESCRIPTION
**Pull request by WimVanRoy**
- Added the warm_start functionality to the wrapper
- Added a .gitignore file for the automatically generated files
- Added simple tests for the warm_start functionality

### CODE REVIEW

File: test/warm_start.jl [hunk 1]
```diff
@@ -0,0 +1,49 @@
+using Test
+
+@testset "Warm start" begin
+
+    using QPALM
+    using LinearAlgebra
+    using SparseArrays
+
+    n, m = 2, 3
+    act = 3
```
Comment (lostella) [inline test/warm_start.jl]: `act` can probably be removed here

**Review by lostella — APPROVED** [review]
Looks good to me, thanks.

We could test that the warm starting actually takes place, by e.g. initializing the iterates from a previously computed solution and checking that 0 iterations take place.

But this already looks fine.

**Comment by WimVanRoy** [comment]
Ok, I improved the testing but it turns out that if you re-create the model and reuse x & y, you only need less iterations (f.ex. from 10 -> 5). This is probably due to the stop criteria that are updated each cycle. 
Only when you can reuse the same model-data, the number of iterations are equal to 1.

If you are happy with the changes, you may merge this branch...

File: test/warm_start.jl [hunk 2]
```diff
@@ -0,0 +1,53 @@
+using Test
+
+@testset "Warm start" begin
+
+    using QPALM
+    using LinearAlgebra
+    using SparseArrays
+
```
Comment (lostella) [inline test/warm_start.jl:9]: It’s better to do

```suggestion
using Random

Random.seed!(0)
```

here, to make this test less shaky.

**Comment by lostella** [comment]
Just fix the random seed, then we can merge. Thank you!

**Review by lostella — CHANGES_REQUESTED** [review]

File: test/warm_start.jl (hunk 2 above)
Comment (WimVanRoy) [reply test/warm_start.jl:9]: I do agree, thank you for the comment!

**Pull request merged**